### PR TITLE
Update diagram and readme

### PR DESCRIPTION
**Pull request by lukesleeman**
This PR updates the readme with some more changes to the design and process

## Changes

- Added updated flow diagram and screen designs
- Tweaks to process, particularly about who is assigned to a particular task
- Added in some developer pro-tips

## Screenshots

Checkout our new protips:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,10 +130,24 @@ flutter run adb
 
 ## Contributing
 
-Check our [issues page](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues) for tasks to work on.
+Check our [issues page](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues?q=is%3Aopen+is%3Aissue+label%3A%22Ready+for+Dev%22) `ready for dev` issues to work on.  Look for issues that are not assigned to somebody.  If you are working on an issue assign yourself to it.  To work on something you must be assigned on the issue. If someone already is, ask if they want help and if they agree add yourself to the assignment. Nobody no in the assignment list should be working on an issue.
```
Comment (maks) [inline README.md:133]: @lukesleeman could you pleae also add here under contributing:

Before you submit a PR, please **ENSURE** that you have run the following commands locally and all the steps pass:
```
flutter clean
flutter format
flutter analyze
flutter test
```
You may want to add a git `pre-commit` hook to do this, see here for more information on [how to use git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)

File: README.md [hunk 2]
```diff
@@ -38,9 +38,7 @@ Once they have done their ✨ magic ✨ the issue should be labeled as 'Ready fo
 
 ⬇️
  
-👩‍💻👨‍💻 **Developers** pick up [ready for Dev issues](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues?q=is%3Aopen+is%3Aissue+label%3A%22Ready+for+Dev%22+) and submit PRs to the project when then get approved and merged.  If you are working on developing an issue, be sure to assign yourself to it, or mention that you are working on it.
- 
-If your committing changes to the docs only add `[skip ci]` to your commit message to save build time.
+👩‍💻👨‍💻 **Developers** pick up [ready for Dev issues](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues?q=is%3Aopen+is%3Aissue+label%3A%22Ready+for+Dev%22+) and submit PRs to the project when then get approved and merged.  If you are working on developing an issue, be sure to assign yourself to it.  If you wwant to work on an issue and someone already is, ask if they want help.  If they agree add yourself to the assignment.
```
Comment (maks) [inline README.md]: typo; wwant

**Review by maks — CHANGES_REQUESTED** [review]
:+1: some small feedback items

File: README.md (hunk 1 above)
Comment (lukesleeman) [reply README.md:133]: Thanks for the suggestion!  I've added it into the readme and will look into using git hooks

File: README.md (hunk 2 above)
Comment (lukesleeman) [reply README.md]: Thanks for that, done!

**Review by maks — APPROVED** [review]
:+1:

**Pull request merged**